.github/workflows/release-pr.yml

```diff
@@ -0,0 +1,69 @@
+name: Create Release PR
+on:
+  create:
+    branches:
+    - 'release/v**'
+
+jobs:
+  build:
+    name: Create Release PR
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Calculate Tag
+      run: |
+        t=$(echo ${{ github.ref }} | sed -ne 's/.*\([0-9]\+\.[0-9]\+\.[0-9]\+\).*/\1/p')
+        if [ -z "$t" ]; then
+        echo Malformed release branch name; failed to extract semVer tag
+        return 1
+        fi
+        echo ::set-env name=TAG_NAME::"v${t}"
+
+    - name: Install release-notes
+      run: |
+        mkdir -p $HOME/deps
+        curl -Lo $HOME/release-notes.tar.gz https://github.com/gsquared94/release-notes/releases/download/v0.1.2/release-notes-linux-amd64.tar.gz
+        tar xfz $HOME/release-notes.tar.gz -C $HOME/deps
+        echo ::add-path::$HOME/deps
+
+    - name: Generate Changelog
+      run: |
+        release-notes GoogleContainerTools skaffold --token=${{ secrets.GITHUB_TOKEN }} > $HOME/change.txt
+
+    - name: Sync examples
+      run: |
+        EXAMPLES_DIR=${PWD}/examples
+        INTEGRATION_EXAMPLES_DIR=${PWD}/integration/examples
+        rm -rf ${EXAMPLES_DIR} && rm -rf ${INTEGRATION_EXAMPLES_DIR}/bazel/bazel-* && cp -r ${INTEGRATION_EXAMPLES_DIR} ${EXAMPLES_DIR} && rm -rf ${EXAMPLES_DIR}/test-*
```

Comment:
The following will wipe out the integration tests as part of the commit, won't it?  Shouldn't it be done in `EXAMPLES_DIR` after the copy?
```
rm -rf ${INTEGRATION_EXAMPLES_DIR}/bazel/bazel-*
```

Reply:
This is the same command being run in `release.sh` today. It's probably for deleting these folders that get created when running this example locally:
![image](https://user-images.githubusercontent.com/39389231/86044929-abe60900-b9ff-11ea-9eb5-0bd25bf9d0c1.png)

That's redundant here since these folders don't exist on master today and are also added to `gitignore`. But I don't see any harm in leaving it here just to be sure.

Reply:
Ohhh, that makes sense.

Reply:
It's worth a comment though